src/__test__/index.test.tsx

```diff
@@ -0,0 +1,12 @@
+// __tests__/index.test.jsx
+
+import { render } from "@testing-library/react";
+
+import "@testing-library/jest-dom";
+import Home from "@/pages/index";
+
+describe("Render Main Page Successfully", () => {
+  it("Renders Home Page Successfully", () => {
+    render(<Home countryCodes={[]} />);
+  });
+});
```

Comment:
This is correct, but I believe the description is kind of redundant. I think it's better to reword it to be something like:

```suggestion
describe("<Home />", () => {
  it("renders home page successfully", () => {
    render(<Home countryCodes={[]} />);
  });
});
```

We can use either proper case or all in lowercase (I think it's Jest convention, but it depends on the project as well). It depends on your choice.